src/Microsoft.Azure.ServiceBus/Message.cs

```diff
@@ -211,9 +200,13 @@ public TimeSpan TimeToLive
 
         // TODO: Calculate the size of the properties and body
         /// <summary>
-        /// Gets the total size of the message in bytes.
+        /// Gets the total size of the message body in bytes.
```

Comment:
What do you suggest? What remark would do good?
Current thought process - Its better to have a public property which doesn't really do anything useful, whose implementation can later be improved with the fix of #109; than make this internal and then later introduce a new property when we have a fix. 
Hence, this change where I explicitly say for now that the size is just the body size.